src/hooks/useAuth.tsx

```diff
@@ -32,6 +47,11 @@ export const AuthContextProvider = ({
   const [isLoggedIn, setIsLoggedIn] = useState<boolean>(false);
   const [loading, setLoading] = useState<boolean>(true);
 
+  // TODO: rename to refreshHandler and use (if needed) in appState changes
```

Comment:
This TODO is handled in the [next PR](https://github.com/lifeomic/react-native-sdk/pull/55)